src/libs/API.js

```diff
@@ -153,7 +159,8 @@ Network.registerResponseHandler((queuedRequest, response) => {
         // There are some API requests that should not be retried when there is an auth failure like
         // creating and deleting logins. In those cases, they should handle the original response instead
         // of the new response created by handleExpiredAuthToken.
-        if (queuedRequest.data.doNotRetry || unableToReauthenticate) {
+        const shouldRetry = lodashGet(queuedRequest, 'data.shouldRetry', true);
```

Comment:
Wouldn't it be safer to assume `false` as the default here?

Reply:
Most things should be retried and the exception is when things pass `shouldRetry: false` (previously this was `doNotRetry`). Thinking more on this.. maybe a better way to handle this would be to add `shouldRetry: true` as the default whenever a request is queued. That way we don't have to have to use `lodashGet()` in the two places.

Reply:
Ah ok good to know, I didn't know we considered it safe to consider that any command is retriable by default, because on other layers we consider it's not safe to retry unless the command is explicitly known to be idempotent.